README.md

```diff
@@ -362,91 +371,51 @@ Downloading cf-standard-names table version 35 from: http://cfconventions.org/Da
 
 ## Installation
 
-### Non-Python dependencies
+### Conda users
```

Comment:
Divided into two sections `conda` and `pip`. Both the `Ubuntu` instructions and how to install `conda` where added as links for further reading.